### PR TITLE
Add twital bundle recipe

### PR DESCRIPTION
**Pull request by goetas**
| Q             | A
| ------------- | ---
| License       | MIT

This is a recipe for https://github.com/goetas/twital-bundle

A fully compatible Twig template engine with enhanced syntax.

### CODE REVIEW

File: goetas/twital-bundle/1.2/config/twital.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+framework:
+    templating:
+        engines: ['twig', 'twital']
+# optional configurations for file extension matching
```
Comment (Nyholm) [inline goetas/twital-bundle/1.2/config/twital.yaml]: Not sure we should include this optional config. I would much rather see a link to the bundle's documentation

**Comment by goetas** [comment]
Hi! Thanks for keeping an eye on this!  Now it is ready! 

(Was not easy! I feel that since all the dependencies become optional, it is not easy to track what is available and what is not).

File: goetas/twital-bundle/1.3/config/packages/twital.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+goetas_twital:
+    full_twig_compatibility: true # see http://twitalbundle.readthedocs.io/en/latest/#full-twig-compatibility
+
+## add here your custom extension handlers, by default Twital parses *.html.twital, *.xml.twital and *.xhtml.twital files
```
Comment (goetas) [inline goetas/twital-bundle/1.3/config/packages/twital.yaml]: @Nyholm If you think is a better idea is can completely remove this comment and let the users find out this detail via documentation.

Was thinking to put just a note "By default Twital parses *.html.twital, *.xml.twital and *.xhtml.twital files, more info about it on http://twitalbundle.readthedocs.io/en/latest/#source-adapters"

**Comment by goetas** [comment]
@Nyholm anything missing with this PR that I should improve?

**Comment by goetas** [comment]
Thanks!

**Pull request merged**